### PR TITLE
[relates #109] Some fixes and include filter step in SF TW scenario

### PR DESCRIPTION
**Pull request by tplevko**

### CODE REVIEW

File: rest-tests/pom.xml [hunk 1]
```diff
@@ -29,6 +29,7 @@
 		<openshift.version>5.6.0.Final</openshift.version>
 		<selenide.version>4.5.1</selenide.version>
 		<guava.version>21.0</guava.version>
+		<jackson.version>1.9.13.redhat-4</jackson.version>
```
Comment (dsimansk) [inline rest-tests/pom.xml]: I'd better to use upstream version from Central.

File: rest-tests/src/test/resources/filters/tw-sf-filter.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[{
+		"path": "text",
+		"op": "contains",
+		"value": "#backendTest"
+	}]
```
Comment (dsimansk) [inline rest-tests/src/test/resources/filters/tw-sf-filter.json]: Lets try to generate filter step from code. Either by backend call or directly from model?

**Comment by dsimansk** [comment]
Pls upgrade syndesis.version to released one from Central also :)

**Comment by tplevko** [comment]
Fixed issues, review this please one more time.

File: rest-tests/src/test/java/io/syndesis/qe/rest/utils/FilterRulesBuilder.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+/**
+ * Used to generate filter rules: "[{ \"path\": \"text\", \"op\": \"contains\", \"value\": \"#backendTest\" }]"
+ *
+ * Nov 6, 2017 Red Hat
+ *
+ * @author tplevko@redhat.com
+ */
+@Slf4j
+public class FilterRulesBuilder {
```
Comment (dsimansk) [inline rest-tests/src/test/java/io/syndesis/qe/rest/utils/FilterRulesBuilder.java]: 👍  💯

File: rest-tests/src/test/resources/mappings/twitter-salesforce.json [hunk 4]
```diff
@@ -1,107 +1,111 @@
...
-    "lookupTables": {
-      "lookupTable": []
-    },
-    "properties": {
-      "property": []
-    }
-  }
-}
-
+	"AtlasMapping": {
```
Comment (dsimansk) [inline rest-tests/src/test/resources/mappings/twitter-salesforce.json]: Lets discuss with @mmelko how to get rid off this json in future :)

Comment (tplevko) [reply rest-tests/src/test/resources/mappings/twitter-salesforce.json]: ok, I will talk with him about possible improvements

**Pull request merged**